### PR TITLE
Astro 983 update button compliance

### PR DESCRIPTION
**Pull request by Design-Kat**
Added compliance language to compliance.json
Added new rule of thumbs based on compliance language to button.md
https://rocketcom.atlassian.net/browse/ASTRO-983

### CODE REVIEW

**Comment by Design-Kat** [comment]
@dmcalester @Cheshire89 - Just finished this up, though I don't seem to have permission to add reviewers. Can @amalyah be added as well please?

File: _content/_data/compliance.json [hunk 1]
```diff
@@ -142,6 +142,11 @@
               "numeral": 8,
               "rule": "Outline Button style shall indicate the less preferred option in grouped Buttons.",
               "status": "current"
+            },
+            {
+              "numeral": 9,
+              "rule": "Primary buttons indicating preferred user actions shall be placed to the right of buttons used for secondary user actions when in Button groups.",
```
Comment (amalyah) [inline _content/_data/compliance.json:148]: Needs consistent use of capitalization of buttons/Buttons. (Uses both here)

File: _content/components/button.md [hunk 2]
```diff
@@ -26,7 +26,8 @@ Action Buttons allow users to trigger actions by clicking, tapping, or pressing
 - Resize button width to accommodate the title; do not abbreviate or truncate button titles.
 - Don’t use an outside label to introduce a button. Instead, clearly title the button.
 - Add an ellipsis (…) to the button title if it opens another window, Dialog Box or app.
-- Within Dialog Boxes, right-justify buttons.
+- In button groups, the primary button with the preferred user action shall be placed on the right and all buttons with secondary actions to the left of the preferred action button.
+- Position buttons consistently across the application. Unless there is a good reason not to, right-align buttons in Astro applications.
```
Comment (amalyah) [inline _content/components/button.md:30]: Could you update the "Do" terminology below to also use the term "right-align" instead  of 
"right justify"?

File: _content/_data/compliance.json [hunk 3]
```diff
@@ -142,6 +142,11 @@
               "numeral": 8,
               "rule": "Outline Button style shall indicate the less preferred option in grouped Buttons.",
               "status": "current"
+            },
+            {
+              "numeral": 9,
```
Comment (amalyah) [inline _content/_data/compliance.json:147]: Does the version number of the compliance page need to be updated with this new change?

File: _content/components/button.md (hunk 2 above)
Comment (amalyah) [reply _content/components/button.md:30]: Also bonus points if you change the first Do/Don't from "User buttons" to "Use buttons". (Won't hold up approval  though)

**Review by amalyah — CHANGES_REQUESTED** [review]

**Comment by amalyah** [comment]
@Design-Kat Can you update the description of the branch with the link to the Jira ticket?

File: _content/_data/compliance.json (hunk 3 above)
Comment (Design-Kat) [reply _content/_data/compliance.json:147]: @dmcalester Do I need to update the version number with the update?

Comment (Design-Kat) [reply _content/_data/compliance.json:147]: @amalyah updated everything except for this, will wait on Duncan's reply.

**Review by amalyah — APPROVED** [review]
Approved pending the version number change. Also, as long as the "This branch is out-of-date with the base branch" warning I'm seeing in the checks list is resolved.

File: _content/_data/compliance.json (hunk 3 above)
Comment (dmcalester) [reply _content/_data/compliance.json:147]: @Design-Kat @amalyah yes, version will need to update. We can do it just prior to launch. https://rocketcom.atlassian.net/browse/ASTRO-1273

**Review by dmcalester — APPROVED** [review]

**Pull request merged**